### PR TITLE
Use file service in notebook content manager

### PR DESCRIPTION
**Pull request by anthonydresser**
Refactors the notebook content manager to use file service of pfs.

### CODE REVIEW

File: src/sql/workbench/parts/notebook/test/node/contentManagers.test.ts [hunk 1]
```diff
@@ -40,7 +46,23 @@ function verifyMatchesExpectedNotebook(notebook: nb.INotebookContents): void {
 }
 
 suite('Local Content Manager', function (): void {
-	let contentManager = new LocalContentManager();
+	let contentManager: LocalContentManager;
+
+	setup(() => {
```
Comment (kevcunnane) [inline src/sql/workbench/parts/notebook/test/node/contentManagers.test.ts]: We've verified all tests pass after this?

**Comment by anthonydresser** [comment]
I don't suspect any behavior or functional changes will happen from this, as behind the scenes its still using pfs, just going through an interface. But certainly people more familiar with the area should test corner cases I would know about. I opened a file, edited a file, saved a file, saved an untitled file and didn't see any problems.

File: src/sql/workbench/parts/notebook/test/node/contentManagers.test.ts (hunk 1 above)
Comment (anthonydresser) [reply src/sql/workbench/parts/notebook/test/node/contentManagers.test.ts]: Well these are unit tests so they run in the pipeline.

**Comment by anthonydresser** [comment]
@kevcunnane @chlafreniere is this fine to merge in?

**Comment by chlafreniere** [comment]
> @kevcunnane @chlafreniere is this fine to merge in?

Go for it. If there are issues we will address tomorrow.

**Comment by kevcunnane** [comment]
@anthonydresser maybe wait until tomorrow as we have some release activities happening, and I'd prefer not to have things unexpectedly broken. Let's check in tomorrow & test the build given the higher scrutiny on Insiders releases for the next week or so.

**Comment by kevcunnane** [comment]
Right, you should be good to check in once we have the Insiders build running now ready & out the door. Thanks for waiting.
